stanza/message.go

```diff
@@ -8,13 +8,29 @@ import (
 // ============================================================================
 // Message Packet
 
+// Subject is an element of a message
+type Subject struct {
+	XMLName xml.Name `xml:"subject"`
+
+	Content string `xml:",chardata"`
+	Lang    string `xml:"lang,attr,omitempty"`
+}
+
+// Body is an element of a message
+type Body struct {
+	XMLName xml.Name `xml:"body"`
+
+	Content string `xml:",chardata"`
+	Lang    string `xml:"lang,attr,omitempty"`
+}
```

Comment:
Can actually combine these two into one. Since the XML element name is described by the `Message` struct (i.e. `xml:"subject,omitempty"`) we can drop it from this definition, making it useable anywhere we want a localised string.

```suggestion
// LocalizedString is a string node with a language attribute.
type LocalizedString struct {
	Content string `xml:",chardata"`
	Lang    string `xml:"lang,attr,omitempty"`
}
```

Reply:
👍